examples/hitl/rearrange_v2/metrics.py

```diff
@@ -0,0 +1,29 @@
+from typing import TYPE_CHECKING, Optional
+
+if TYPE_CHECKING:
+    from habitat_hitl.app_states.app_service import AppService
+
+
+class Metrics:
```

Comment:
IMO a plural word is a bad choice for a class name. E.g. what if one day you need a list of these, what would you call that variable? `metricses`?

Reply:
We can rename the class.

I will do this separately as a refactor pass.